circleci/docker-publish

```diff
@@ -59,12 +59,14 @@ check_ecr_vars() {
 }
 
 ecr_login(){
+  REGION=$1
```

Comment:
looks like this file prefers tabs to spaces :) 
```suggestion
	REGION=$1
```

Reply:
edit: maybe just this function 🤔

Reply:
weird lol, I'll just change the whole function